dashboard/lib/config.js

```diff
@@ -14,83 +14,84 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-'use strict';
 
+'use strict';
 
-var prog = require('../package.json'),
+var path = require('path'),
+    prog = require('../package.json'),
+    name = path.basename(process.argv[1], '.sh'),
     logger = require('./logger'),
-    fs = require('fs'),
-    util = require('util'),
     optimist = require('optimist'),
-    yamljs = require('yamljs');
+    yamljs = require('yamljs'),
+    util = require('util'),
+    fs = require('fs');
+
 
 /**
  * Program configuration.
  *
  * <var>config</var> attributes will be updated with those read from <var>config_file</var>, if exists,
- * from environment variables and from command line arguments, in that order.
- *
- * @type {{config_file: *, log_level: string}} default configuration file.
+ * and from command line arguments, in that order.
  */
-
-var
-    config = {
-        config_file: util.format(__dirname + '/../config/%s.yml', prog.name),
-        log_level: 'DEBUG',
-        secret:'ssshhh',
-        cbroker: {
-            host: '',
-            port: '',
-            path: '/NGSI10/queryContext'},
-        idm: {
-            host: 'account.lab.fiware.org',
-            clientId: '',
-            clientSecret: '',
-            url: 'https://account.lab.fiware.org',
-            callbackURL: 'http://fi-health.lab.fiware.org/login'
-        },
-        mailman: {
-            host: 'localhost',
-            port: '8000',
-            path: '/',
-            email_from: ''
-        },
-        paths: {
-            reports_files: '/var/www/html/RegionSanityCheck',
-            reports_url: '/check/report'
-        }
-    };
-
+var config = {
+    config_file: path.join(__dirname, '/../config', name + '.yml'),
+    log_level: 'DEBUG',
+    listen_port: 3000,
+    secret:'ssshhh',
+    paths: {
+        reports_url: '/check/report',
+        reports_files: '/var/www/html/RegionSanityCheck'
```

Comment:
Currently, we are moving generated reports to `/var/www/html/RegionSanityCheckv2`. If we configured this new path, we must change Jenkins configuration.